src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
```

Comment:
https://web-beta.archive.org/web/20141129025434/https://technet.microsoft.com/en-us/library/security/cc750357.aspx which was last updated May 2014 has the same language:

> Note – There is no enforcement of the FIPS policy by the operating system or the validated cryptographic modules. Instead, each individual application must check this flag and enforce 

It seems .NET at 4.6.2 is supposed to throw an exception when using SHA256Managed at https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/sha256managed.cs,35, but it doesn't throw it. As I see it, .Net has been written "to abide by the FIPS security policy" but doesn't actually abide by it.

Or is my logic wrong here? Help appreciated! Thanks....